.github/workflows/ci.yaml

```diff
@@ -40,7 +40,7 @@ jobs:
       - uses: actions/upload-artifact@v2
         with:
           name: build
-          path: build
```

Comment:
Have we confirmed that this change works without changing the `build-folder` property in `deploy-preview` and `deploy` steps?

Reply:
For `deploy-preview` yes because the preview link works and the github bot posts the preview link on every push to this branch https://github.com/threshold-network/splash-page/pull/13#issuecomment-1139101896. So I believe should be OK with the `deploy` step.